app/feedme/websocket/realtime_manager.py

```
@@ -0,0 +1,571 @@
+            if not self.heartbeat_task:
+                self.heartbeat_task = asyncio.create_task(self._heartbeat_loop())
+            
+            logger.info(f"WebSocket connected: user={user_id}, room={room_id}")
+            return connection_info
+            
+        except Exception as e:
+            logger.error(f"Failed to connect WebSocket: {e}")
+            try:
+                await websocket.close()
+            except:
+                pass
```

Ruff (0.11.9)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)